modules/components/org.wso2.carbon.appfactory.core/src/main/java/org/wso2/carbon/appfactory/core/workflow/TenantCreationDefaultWorkflowExecutor.java

```diff
@@ -0,0 +1,108 @@
+/*
+ *   Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This is fixed in following commit
https://github.com/wso2/product-af/pull/664/commits/6ddbe7623eeb6fdc99f0e16cadec68bdb1ffe0ea